### DISCUSSION TITLE
The thumb query  parameter does not work for me v0.8.0 and Minio

### DISCUSSION THREAD

**Discussion by hopsersmerk** [pocketbase/pocketbase#1105] [Q&A] [answered]
I'm trying to crop an image with the query param thumb but I can't get it to work.
Am I doing it right?

[https://pb.hopsersmerk.com/api/files/bxztvcgeu9s6p9y/hnwyzs94ttnszu2/photo_1502759683299_cdcd6974244f_JZ54RWiTgO.jpg?thumb=100x300](url)

![Imagen](https://pb.hopsersmerk.com/api/files/bxztvcgeu9s6p9y/hnwyzs94ttnszu2/photo_1502759683299_cdcd6974244f_JZ54RWiTgO.jpg?thumb=100x300)

I am using version **0.8.0** and i am storing the files in **minio**

**Comment by ganigeorgiev** [comment, marked as answer]
Make sure that the thumb size(s) are listed in your `file` "Thumb sizes" field option:

![file_field](https://i.imgur.com/mfU1k0c.png)

**Reply by hopsersmerk** [reply]
I did not have that section configured, now it is working correctly! Thanks a lot! 

[https://pb.hopsersmerk.com/api/files/bxztvcgeu9s6p9y/hnwyzs94ttnszu2/photo_1502759683299_cdcd6974244f_JZ54RWiTgO.jpg?thumb=300x300](url)

![Imagen](https://pb.hopsersmerk.com/api/files/bxztvcgeu9s6p9y/hnwyzs94ttnszu2/photo_1502759683299_cdcd6974244f_JZ54RWiTgO.jpg?thumb=300x300)
